uk.co.compucorp.civicrm.hrcore/config/menu/main.php

```diff
@@ -1,5 +1,17 @@
 <?php
 
+$result = civicrm_api3('OptionValue', 'getsingle', [
+  'option_group_id' => 'custom_search',
+  'name' => 'CRM_HRCore_Form_Search_StaffDirectory',
+]);
+
+$searchDirectoryURL = '';
+if (!empty($result['value'])) {
+  $searchDirectoryURL =
+    "civicrm/contact/search/custom?csid={$result['value']}&force=1&reset=1&select_staff=current";
+}
+
+
```

Comment:
@tunbola let's avoid adding logic to configuration files. If a menu item requires some logic, then it should be a class